extensions/wikia/WikiaHubsServices/models/EditHubModel.class.php

```diff
@@ -614,11 +614,20 @@ private function getFileUrl($fileName, File $file) {
 	private function getFileCityId(File $file) {
 		global $wgCityId;
 
-		$dbName = $file->getRepo()->getDBName();
-		$wiki = WikiFactory::getWikiByDB( $dbName );
-
-		if ( !empty ( $wiki ) ) {
-			return $wiki->city_id;
+		$repo = $file->getRepo();
+
+		/**
+		 * If file is from WikiaLocalRepo then city id
+		 * will equal to wiki from current context
+		 *
+		 */
+		if ( !( $repo instanceof WikiaLocalRepo ) ) {
```

Comment:
Note, that `getDBName()` only exists in `extensions/wikia/VideoHandlers/filerepo/WikiaForeignDBViaLBRepo.class.php`. This check therefore has a leak.